README.md

```diff
@@ -33,6 +33,19 @@ cd <DESTINATION>/web
 - Start the web server with `../bin/drush runserver`. This defaults to `127.0.0.1:8888`, you can change this by appending a new host and port, e.g. `../bin/drush runserver local.contentacms.io:8000`
 - Generate a one-time login link `../bin/drush user-login --uri="http://127.0.0.1:8888"`
 
+### CURL
+
+Once your site is running locally, you might want to use Curl to examine the pre-installed content:
+
+```bash
+curl --header 'Accept: application/vnd.api+json' http://127.0.0.1:8888/api/recipes
+```
+
+The result will be a list of recipes. Note that:
+
+- The Drupal implementation of the jsonapi uses entity uuids to identify individual content resources. Append a `/uuid` to the URL above to fetch a single recipe. 
```

Comment:
Maybe:

Append a `/{{uuid}}` to the URL above to fetch a single recipe.

To avoid people being too literal.